src/Components/Shifting/models.ts

```diff
@@ -8,7 +8,7 @@ export interface IShift {
   emergency: boolean;
   status: string;
   origin_facility_object: FacilityModel;
-  origin_facility: string;
+  origin_facility: string | number;
```

Comment:
Why is this being typed as a number?

```suggestion
  origin_facility: string;
```